.kitchen.yml

```diff
@@ -1,16 +1,18 @@
 ---
 driver:
-  name: docker
-  use_sudo: false
-  dns:
-    - 8.8.8.8
-    - 8.8.4.4
+  name: dokken
+  chef_version: latest
 
 driver_config:
   socket: <%= ENV['DOCKER_HOST'] || 'unix:///var/run/docker.sock' %>
+  use_internal_docker_network: true
+  privileged: true
```

Comment:
I know we were testing this for the docker version of this build -- do we need to set "privileged: true" for the machine version?

Reply:
Yes, apparently this is needed so that kitchen can start the cron service.